test/v2/services/CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionServiceSpec.scala

```diff
@@ -67,7 +66,20 @@ class CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionServiceSpec extends Uni
 
           await(service.amend(Request)) shouldBe Left(ErrorWrapper(correlationId, error))
         }
-      downstreamErrorMap.foreach(args => (serviceError _).tupled(args))
+
+      Seq(
+        "INVALID_NINO"           -> NinoFormatError,
+        "INVALID_TYPE"           -> InternalError,
+        "INVALID_TAX_YEAR"       -> TaxYearFormatError,
+        "INVALID_PAYLOAD"        -> InternalError,
+        "INVALID_CORRELATIONID"  -> InternalError,
+        "NOT_FOUND_PROPERTY"     -> NotFoundError,
+        "NOT_FOUND"              -> NotFoundError,
+        "GONE"                   -> InternalError,
+        "TAX_YEAR_NOT_SUPPORTED" -> RuleTaxYearNotSupportedError,
```

Comment:
Should this be **RuleHistoricTaxYearNotSupportedError** instead of **RuleTaxYearNotSupportedError** ?